packages/map/src/ClickOutside.re

```diff
@@ -0,0 +1,29 @@
+open Webapi.Dom;
+
+let handleClickOutside = (domElement: Dom.element, e: Dom.mouseEvent, fn) => {
```

Comment:
If you want some cleaner code, you can remove all explicit types like `Dom.element`. The compiler is powerful enough to infer them correctly!

Reply:
Awesome! I removed them.